src/device-manager/java/WeaveDeviceManager-JNI.cpp

```diff
@@ -184,6 +184,8 @@ extern "C" {
     NL_DLL_EXPORT jlong Java_nl_Weave_DataManagement_WdmClientImpl_newDataSink(JNIEnv *env, jobject self, jlong wdmClientPtr, jobject resourceIdentifierObj, jlong profileId, jlong instanceId, jstring path);
     NL_DLL_EXPORT void Java_nl_Weave_DataManagement_WdmClientImpl_beginFlushUpdate(JNIEnv *env, jobject self, jlong wdmClientPtr);
     NL_DLL_EXPORT void Java_nl_Weave_DataManagement_WdmClientImpl_beginRefreshData(JNIEnv *env, jobject self, jlong wdmClientPtr);
+    NL_DLL_EXPORT void Java_nl_Weave_DataManagement_WdmClientImpl_beginFetchEvents(JNIEnv *env, jobject self, jlong wdmClientPtr, jint timeoutSec);
+    NL_DLL_EXPORT jstring Java_nl_Weave_DataManagement_WdmClientImpl_getEvents(JNIEnv *env, jobject self, jlong wdmClientPtr);
```

Comment:
here you are using Java_nl_Weave_DataManagement_WdmClientImpl_getEvents, but in line 4019, you are using jstring Java_nl_Weave_DataManagement_WdmClientImpl_getEventsNative(JNIEnv *env, jobject self, jlong wdmClientPtr)

the function name should be same.